libs/controller/controller.py

```diff
@@ -710,18 +743,26 @@ def get_latest_build(self, model=None):
 
 class FirmwareUtility(JFrogUtility):
 
-    def __init__(self, sdk_client=None, jfrog_credentials=None, controller_data=None, customer_id=None):
+    def __init__(self,
+                 sdk_client=None,
+                 jfrog_credentials=None,
+                 controller_data=None,
+                 customer_id=None,
+                 model=None,
+                 version=None):
         super().__init__(credentials=jfrog_credentials)
         if sdk_client is None:
             sdk_client = Controller(controller_data=controller_data, customer_id=customer_id)
         self.sdk_client = sdk_client
         self.firmware_client = FirmwareManagementApi(api_client=sdk_client.api_client)
         self.jfrog_client = JFrogUtility(credentials=jfrog_credentials)
         self.equipment_gateway_client = EquipmentGatewayApi(api_client=sdk_client.api_client)
+        self.model = model
+        self.fw_version = version
 
-    def get_latest_fw_version(self, model="ecw5410"):
+    def get_fw_version(self):
```

Comment:
whats the difference betwen latest_build and latest_fw_version?

Reply:
get_latest_build is part of jfrog utility that is inherited by firmware utility which contains get_fw_version,
i used this so in order to seperate the two entity - Controller and jfrog

Reply:
we can remove this function as we can directly use the jfrog one